### PR TITLE
Z-Wave: Reword firmware warnings, add Z-Station as recommended

### PR DESCRIPTION
**Pull request by AlCalzone**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Revised documentation for Z-Wave controllers to clarify firmware recommendations.
	- Provided specific guidance on preferred firmware versions for both 700 and 800 series controllers.
	- Emphasized the importance of upgrading firmware for enhanced stability and preventing unresponsiveness.

### CODE REVIEW

File: source/_docs/z-wave/controllers.markdown [hunk 1]
```diff
@@ -9,9 +9,22 @@ You need to have a compatible Z-Wave stick or module installed. The following de
 
 {% warning %}
 
-Until recently, 700 series Z-Wave Controllers had a bug that could cause the mesh to be flooded on some networks and the controller to become unresponsive. At present, all 700 series controllers share the same firmware and are subject to this bug. It appears that this bug is largely, if not completely, resolved as of firmware version 7.17.2.
-
-Users should upgrade the firmware on all 700 series controllers to version 7.17.2 or greater. Firmware can be upgraded using the below directions:
+The firmwares of 700 and 800 series Z-Wave controllers have several bugs which impact the stability of the mesh and can cause the controller to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
+- 700 series:
+  - prefer SDK versions 7.17.2 to 7.18.x
```
Comment (frenck) [inline source/_docs/z-wave/controllers.markdown:15]: Should we avoid the term SDK here and replace it with firmware version to make it less technical for the reader?

Comment (AlCalzone) [reply source/_docs/z-wave/controllers.markdown:15]: I'd love to, but manufacturers have their own firmware versioning that's not related to the SDK versions.
e.g. Zooz has 1.10, 1.20, 1.30, 1.40. Z-Wave.me seems to have 7.x firmwares, but with different minor versions than the SDK.

Comment (AlCalzone) [reply source/_docs/z-wave/controllers.markdown:15]: Unless we want to explicitly list the recommended firmware versions, but the warning may span half a page then.

Comment (frenck) [reply source/_docs/z-wave/controllers.markdown:15]: Hmm it might help to explain just to the reader that in that case.

Comment (AlCalzone) [reply source/_docs/z-wave/controllers.markdown:15]: Will add an explanation

File: source/_docs/z-wave/controllers.markdown [hunk 2]
```diff
@@ -9,9 +9,24 @@ You need to have a compatible Z-Wave stick or module installed. The following de
...
+  - avoid SDK versions before 7.17.2
+  - avoid SDK versions 7.20 to 7.21.3
+ 
+- 800 series
+  - prefer SDK versions 7.22.x
+  - SDK versions 7.17.2 to 7.19.x are okay
+  - avoid SDK versions before 7.17.2
+  - avoid SDK versions 7.20 to 7.21.3
+ 
+Note that the SDK version does not have to match the firmware version. If you are unsure which SDK versions a firmware is based on, contact the manufacturer.
```
Comment (frenck) [inline source/_docs/z-wave/controllers.markdown]:
```suggestion
{% note %}
The SDK version does not have to match the firmware version. If you are unsure which SDK versions a firmware is based on, contact the manufacturer of your device.
{% endnote %}
```

File: source/_docs/z-wave/controllers.markdown [hunk 3]
```diff
@@ -9,9 +9,24 @@ You need to have a compatible Z-Wave stick or module installed. The following de
...
+
+Users should upgrade the firmware on all 700 and 800 series controllers to a recommended version.
```
Comment (frenck) [inline source/_docs/z-wave/controllers.markdown]: If it implies a requirement for success we should wrap it in an important box.

Additionally, small adjustment toward the user from reading it.

```suggestion
{% important %}
You should upgrade the firmware on all 700 and 800 series controllers to a recommended version.
{% endimportant %}
```

**Review by frenck — APPROVED** [review]
Thanks, @AlCalzone 👍 

../Frenck

**Pull request merged**